src/main/java/com/gooddata/featureflag/FeatureFlagService.java

```diff
@@ -0,0 +1,183 @@
+/*
+ * Copyright (C) 2007-2015, GoodData(R) Corporation. All rights reserved.
+ */
+package com.gooddata.featureflag;
+
+import com.gooddata.AbstractService;
+import com.gooddata.GoodDataException;
+import com.gooddata.project.Project;
+import org.springframework.web.client.RestClientException;
+import org.springframework.web.client.RestTemplate;
+
+import java.net.URI;
+
+import static com.gooddata.featureflag.FeatureFlags.AGGREGATED_FEATURE_FLAGS_TEMPLATE;
+import static com.gooddata.featureflag.ProjectFeatureFlag.PROJECT_FEATURE_FLAG_TEMPLATE;
+import static com.gooddata.featureflag.ProjectFeatureFlags.PROJECT_FEATURE_FLAGS_TEMPLATE;
+import static com.gooddata.util.Validate.notEmpty;
+import static com.gooddata.util.Validate.notNull;
+
+/**
+ * Provides feature flag management. Feature flag is a boolean flag used for enabling / disabling some specific feature
+ * of GoodData platform. It can be used in various scopes (per project, per project group, per user, global etc.).
+ */
+public class FeatureFlagService extends AbstractService {
+
+    /**
+     * Constructs service for GoodData feature flags management.
+     *
+     * @param restTemplate RESTful HTTP Spring template
+     */
+    public FeatureFlagService(final RestTemplate restTemplate) {
+        super(restTemplate);
+    }
+
+    /**
+     * Returns aggregated feature flags for given project and current user (aggregates global, project group, project
+     * and user feature flags).
+     * It doesn't matter whether feature flag is enabled or not, it'll be included in both cases.
+     *
+     * @param project project, cannot be null
+     * @return aggregated feature flags for given project and current user
+     */
+    public FeatureFlags getFeatureFlags(final Project project) {
```

Comment:
I tend to name `listFeatureFlags()` here:
# the method used to be called `listAggregatedFeatureFlags()`
# we name listing methods with `list` prefix (eg. WarehouseService.listWarehouses(), ProcessService.listProcesses(),...) and the `FeatureFlags` class is more like collection